lib/livebook/session.ex

```diff
@@ -467,10 +478,12 @@ defmodule Livebook.Session do
         autosave_timer_ref: nil,
         autosave_path: opts[:autosave_path],
         save_task_pid: nil,
-        saved_default_file: nil
+        saved_default_file: nil,
+        system_memory_timer_ref: nil,
+        memory_usage: nil
       }
 
-      {:ok, state}
+      {:ok, state |> system_memory_usage()}
```

Comment:
This will immediately `notify_update`, which we don't want. Instead we can initial `memory_usage` above to `%{runtime: nil, system: Utils.fetch_system_memory()}`